Basic-Car-Maintenance/Shared/Settings/Views/ChooseAppIconView.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  ChooseAppIconView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Daniel Lyons on 10/11/23.
+//
+
+import SwiftUI
+
+struct ChooseAppIconView: View {
+  @State var viewModel: ChooseAppIconViewModel = .init()
+  
+  private func isSelected(_ icon: AppIcon) -> Bool {
+    return viewModel.selectedAppIcon == icon
+  }
+  
+  var body: some View {
+    let columns = [
+      GridItem(.flexible(minimum: 30, maximum: 300)),
+      GridItem(.flexible(minimum: 30, maximum: 300))
+    ]
+    
+    return Form {
```

Comment:
When fixing the comment above, you can then remove this `return`

Reply:
Done.